src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_NUMBERS = 5;
+
+    public Figure getRandomFigure() {
+        Figure figure;
+
+        switch (new Random().nextInt(MAX_NUMBERS) + 1) {
+            case 1:
```

Comment:
what figure must be generated in case 1?

Reply:
refactor the code